src/assets/pages/index.html

```diff
@@ -34,7 +42,7 @@ <h5>{{ product.name | truncate(120) }}</h5>
             <p>
                 {% if product.discount_percentage != 0 %}
                     <span style="text-decoration: line-through;">{{ product.price }}€</span><br>
-                    <span style="color: rgb(227, 43, 43)">{{ product.discounted_price }}€</span> 
+                    <span style="color: rgb(227, 43, 43)">{{ (product.price * (1 - product.discount_percentage)) | round(2) }}€</span> 
```

Comment:
Didn't we already calculate this in the backend?

Reply:
Done